module/icp/algs/blake3/blake3_impl.c

```diff
@@ -0,0 +1,127 @@
+
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License, Version 1.0 only
+ * (the "License").  You may not use this file except in compliance
+ * with the License.
+ *
+ * You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+ * or http://www.opensolaris.org/os/licensing.
+ * See the License for the specific language governing permissions
+ * and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2021 - 2022 Tino Reichardt <milky-zfs@mcmilk.de>
+ */
+
+#include <sys/zfs_context.h>
+#include <sys/zio_checksum.h>
+#include "blake3_impl.h"
+
+static const blake3_impl_ops_t *blake3_impls[] = {
```

Comment:
I don't see why this couldn't be `const`?

Reply:
> I don't see why this couldn't be `const`?

Just `const` instead of `static const` ?

Reply:
No, thank god it's static, but `blake3_impls` isn't const:
```c
 static const blake3_impl_ops_t *const blake3_impls[] = { 
```